server/text_generation_server/models/custom_modeling/flash_llama_modeling.py

```diff
@@ -143,6 +146,22 @@ def forward(self, hidden_states, residual=None):
                 res = hidden_states
 
             return normed_hidden_states, res
+        elif IS_ROCM_SYSTEM:
```

Comment:
@OlivierDehaene, this should be refactored to its own file (like flash-attn, paged attention...) to cleanup the conditional imports.